### DISCUSSION TITLE
ESP32-C3 GPIO19 state on power on?

### DISCUSSION THREAD

**Discussion by freedomlives** [arendst/Tasmota#22460] [Q&A] [answered]
I have a relay board that uses this GPIO for controlling the relay. When I apply power to the board, the relay pulses on and off. I notice that the ESP32-C3 uses GPIO18 & GPIO19 for USB-JTAG. Would these have pullups that would be enabled by default?
In my application of this particular AliExpress relay board, it is just an annoyance for me (streetlights will blink on when the board's power supply is cycled, but I turn off the circuit breakers for the lights when doing this, so no problem), but may be something people will want to be aware of if designing a board (which I plan to do this winter).

**Comment by Jason2866** [comment, marked as answer]
Regarding to the datasheet GPIO 18 has a high level glitch at startup.

**Reply by freedomlives** [reply]
I guess the same thing on GPIO19 then.
